trace-time/src/lib.rs

```diff
@@ -48,8 +46,7 @@ impl PerfTimer {
 impl Drop for PerfTimer {
 	fn drop(&mut self) {
 		let elapsed = self.start.elapsed();
-		let ms = elapsed.subsec_nanos() as f32 / 1_000_000.0 +
-				 elapsed.as_secs() as f32 * 1_000.0;
+		let ms = elapsed.subsec_nanos() as f32 / 1_000_000.0 + elapsed.as_secs() as f32 * 1_000.0;
```

Comment:
use https://doc.rust-lang.org/std/time/struct.Duration.html#method.as_millis instead, introduced in rust 1.33

```suggestion
		let ms = elapsed.as_millis();
```